### ISSUE TITLE
Help of training, can't get 31.4 model

### ISSUE THREAD

**Issue by lampsonSong** [signatrix/efficientdet#43]
Thanks for the great work firstly. 

I run the code successfully, and train the model with coco2017, but the best model I trained can just reach 22% in coco2017 val set. I have tried lr with 1e-4, 1e-5 and 5e-6. Compared with the author's code, I use batch_size = 14 and four TITAN cards. Are there any tips or warnings when training?

**Comment by elv-xuwen** [comment]
> Thanks for the great work firstly.
> 
> I run the code successfully, and train the model with coco2017, but the best model I trained can just reach 22% in coco2017 val set. I have tried lr with 1e-4, 1e-5 and 5e-6. Compared with the author's code, I use batch_size = 14 and four TITAN cards. Are there any tips or warnings when training?

I have the same issue..

**Comment by lxyyang** [comment]
@lampsonSong In paper, lr is 0.08 with 128 batchsize. have you try lr with 0.01?

**Comment by lampsonSong** [comment]
> @lampsonSong In paper, lr is 0.08 with 128 batchsize. have you try lr with 0.01?

yes, I have tried. Cann't get better results. Have you get the same accuracy as this project?

**Comment by lxyyang** [comment]
@lampsonSong  I also tried it，but can not get better result。

**Comment by zylo117** [comment]
> @lampsonSong I also tried it，but can not get better result。

I did, but with a lot of changes of course. 
[Yet-Another-EfficientDet-Pytorch](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch)
